### PR TITLE
breadcrumbs added on course detail page

### PR DESCRIPTION
**Pull request by tasawernawaz**
[ECOM-6046](https://openedx.atlassian.net/browse/ECOM-6046)

**Screenshot:**

screen shot 2017-01-05 at 2 52 47 pm

### CODE REVIEW

**Comment by tasawernawaz** [comment]
@waheedahmed Please review this.

File: course_discovery/static/sass/publisher/course_form.scss [hunk 1]
```diff
@@ -42,3 +42,11 @@
   padding-bottom: 2px;
   padding-top: 2px;
 }
+
+.course-breadcrumb {
```
Comment (waheedahmed) [inline course_discovery/static/sass/publisher/course_form.scss]: Add this in ```publisher.scss``` instead here.

**Review by waheedahmed — APPROVED** [review]
One minor comment otherwise looks good.

**Comment by tasawernawaz** [comment]
@bderusha Please review this.

**Review by bderusha — APPROVED** [review]

**Pull request merged**